packages/components/app/styles/components/pagination.scss

```diff
@@ -4,27 +4,48 @@
 
 @use "../mixins/focus-ring" as *;
 
+// TODO: Move to pagination.json tokens
+:root {
+  --token-pagination-child-spacing-vertical: 8px;
+  --token-pagination-child-spacing-horizontal: 20px;
+}
+
 
 // * Pagination Parent (wrapper) component
 // -------------------------------------------------------------------
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
+
+  // max-width is based on where component layout potentially breaks down
+  @media screen and (max-width: 1000px) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
 }
 
 
 // * Sub-component: Info
 // ---------------------------------------
 
 .hds-pagination-info {
-  grid-area: left;
+  grid-area: info;
   justify-self: flex-start;
   white-space: nowrap;
+
+  @media screen and (max-width: 1000px) {
+    margin:
+      var(--token-pagination-child-spacing-vertical)
```

Comment:
not a blocker, but probably at this point I would do something like this (also for the selector, only on the other side):

```
.hds-pagination-info {
  ...
  margin-right: var(--token-pagination-child-spacing-vertical);

  @media screen and (max-width: 1000px) {
    margin-top: var(--token-pagination-child-spacing-vertical);
  }
}

because in any case, a right margin for the info is useful, also for the non-mobile version, to avoid the "prev" control and the "info" block to touch. what do you think?
```